tests/features/collection/collection_overview.feature

```diff
@@ -0,0 +1,16 @@
+@api
+Feature: Collections Overview
+  Scenario: View collection overview
+    Given collections:
+      | name              | description                    | author          | uri                        |
+      | eHealth           | Supports health-related fields | Arnold Sideways | http://drupal.org/notfound |
+      | Open Data         | Facilitate access to data sets | Madame Sharn    | http://joinup.eu/coll1     |
+      | Connecting Europe | Reusable tools and services    | Ptaclusp IIb    | http://joinup.eu/coll2     |
+    And The collections are indexed
+    When I visit the collection overview page
```

Comment:
Great stuff, this is really good from the BDD perspective, it describes perfectly the intention of the user. In the original test that was proposed in ticket ISAICP-2205, this was saying "When I visit '/collections'" which is not so good, since it implies that users have to browse the site by memorizing URLs.

:+1: